python/lsst/afw/math/statistics.cc

```diff
@@ -86,7 +86,16 @@ PYBIND11_PLUGIN(_statistics) {
 
     py::class_<StatisticsControl, std::shared_ptr<StatisticsControl>> clsStatisticsControl(mod, "StatisticsControl");
 
-    clsStatisticsControl.def(py::init<>());
+    py::enum_<StatisticsControl::WeightsBoolean>(clsStatisticsControl, "WeightsBoolean")
+        .value("WEIGHTS_FALSE", StatisticsControl::WeightsBoolean::WEIGHTS_FALSE)
+        .value("WEIGHTS_TRUE", StatisticsControl::WeightsBoolean::WEIGHTS_TRUE)
+        .value("WEIGHTS_NONE", StatisticsControl::WeightsBoolean::WEIGHTS_NONE)
+        .export_values();
+
+    clsStatisticsControl.def(py::init<double, int, lsst::afw::image::MaskPixel, bool,
+                                      typename StatisticsControl::WeightsBoolean>(),
+                             "numSigmaClip"_a = 3.0, "numIter"_a = 3, "andMask"_a = 0x0, "isNanSafe"_a = true,
+                             "useWeights"_a = StatisticsControl::WEIGHTS_NONE);
```

Comment:
I'm puzzled what happened here. Did you adopt my modernization of the file (the diff suggests not) or just use your version of the file.